### ISSUE TITLE
Sessions Broken Under Safari v10.0.3 Under OS X/macOS v10.11.6 'El Capitan'

### ISSUE THREAD

**Issue by RandomDSdevel** [sessions-extension/Sessions#2]
Some time around when I updated my local copy of Safari to v10.0.3 over my local copy of OS X/macOS v10.11.6 'El Capitan,' Sessions v1.5 has partly stopped working for me.  I can access the extension's toolbar trigger button's click-and-hold popover, but its main session-management window remains beyond my reach.

**Comment by RandomDSdevel** [comment]
I _think_ I remember seeing something about extension handling changing a bit as part of the update in its release notes, but I could be mistaken.  Guess I'd better go double-check…

**Comment by RandomDSdevel** [comment]
Great, now I can't find that non-security changelog I thought I saw earlier.  _Rats!_

**Comment by sessions-extension** [comment]
I have been unable to duplicate this issue in an El Capitan VM, unfortunately. (I did not see any Safari 10.0.3 changes mentioned in the update window aside from the security link.)

If you have any other Safari Extensions installed, particularly content blockers, try temporarily disabling them in Safari Preferences -> Extensions by unchecking the boxes next to their names and restarting Safari to see if that has any effect on this issue.

I am assuming that the "Show Manager" button in the overview popover is similarly unresponsive?

**Comment by RandomDSdevel** [comment]
@sessions-extension:  
> …
>
> I am assuming that the "Show Manager" button in the overview popover is similarly unresponsive?  

That is correct.  However, I was able to get Sessions to show up today, so I've come to the conclusion that it was a latency issue related to some part of the state of Safari's cache.  I didn't have to mess with which extensions I have enabled and disabled either, so now I feel a bit silly for opening this issue in the first place…; heh, heh.
